.buildkite/pipeline.yml

```diff
@@ -1,51 +1,70 @@
-docker-container: &docker-container
-  plugins:
-    - docker#v3.8.0:
-        image: "public.ecr.aws/automattic/android-build-image:v1.1.0"
```

Comment:
Question (❓): Is that `docker` plugin no longer needed?

Cc @oguzkocer

Reply:
👋 @AliSoftware !

Thanks so much for sharing and providing some extra clarity for my benefit. 🙏 

Yes, I was expecting that something like that is happening. I just wanted to make sure of that since I am currently upgrading to `android-build-image:v1.2.0` as part of my `Gradle & AGP` update PRs and I was trying to make sure I know what I need to be doing when a merge conflict pops-up.

🙇

Reply:
@ParaskP7 It's OK to keep upgrading to `v1.2.0` for now since it's a simple change. With `AMI` we will remove Docker plugin completely, but until then, it's still useful.

One option would be to upgrade the projects to `AMI` instead, but I am not a fan of combining a change like that with Gradle upgrade, so my suggestion would be to keep it simple for now.

Reply:
👋 @oguzkocer ! 🙇 

> @ParaskP7 It's OK to keep upgrading to v1.2.0 for now since it's a simple change. With AMI we will remove Docker plugin completely, but until then, it's still useful.

100%, I was just trying to make sure I'll do the correct thing afterwards, that is during a merge conflict, to pick the right resolution. 👍 

> One option would be to upgrade the projects to AMI instead, but I am not a fan of combining a change like that with Gradle upgrade, so my suggestion would be to keep it simple for now.

I agree, I wouldn't do that for the very same reason. 💯